src/utilities/hooks/ModalPaymentContext/types.ts

```diff
@@ -14,13 +15,15 @@ import { BrowsePageSeller } from '../../api/types';
 export type ModalPaymentState = {
   amount: string;
   customInput: boolean;
+  coveredByCustomer: boolean;
   modalView: number;
   sellerData: BrowsePageSeller;
 };
 
 export const defaultState: ModalPaymentState = {
-  amount: '5',
+  amount: '25',
```

Comment:
Why did we update this?

Reply:
@justintmckibben Whoops, I was playing around with defaulting the selected amount and forgot to change it back. I'll revert this on next push.